docker-compose.yml

```diff
@@ -11,10 +18,30 @@ services:
       - isolated-network
     depends_on:
       - mongo
+      - grafana
+      - prometheus
+
   mongo:
     image: mongo
     ports:
       - '27017:27017'
-networks:
-  isolated-network:
-    driver: bridge
+
+  prometheus:
+    image: prom/prometheus
+    container_name: prometheus
+    volumes:
+      - ./prometheus.yml:/etc/prometheus/prometheus.yml
+    ports:
+      - "9090:9090"
+    networks:
+      - isolated-network
+      - default
+
+
+  grafana:
+    image: grafana/grafana:4.5.2
+    ports:
+      - "3000:3000"
```

Comment:
the front end runs on the same port, on the VM.

Reply:
Its now changed to 3001:3001